doc/compodoc_sources/how-to-guides/create-new-entity-type.md

```diff
@@ -1,21 +1,67 @@
 # How to create a new Entity Type
 "Entities" are the objects around which our whole system is built.
-There may for example be User, Child or School classes representing the types of objects the users can view interact with. To extend the system with types, you can create a new module and extend the existing base of the Entity system.
+Entity types specify a data structure for records in the case management system,
+defining both the data object as well as some details how the records are displayed in the UI to users.
+There may for example be User, Child or School classes representing the types of objects the users can view in lists and detail forms and interact with.
 
-The [EntityModule](../../modules/EntityModule.html) specifies the base class and generic features, specific entity types can extend upon it.
-For an example, have a look at the [User](../../classes/User.html#source) class that extends `Entity`.
+To adapt the platform to a certain use case, you usually want to adapt entity types or create new ones.
+This defines what fields users can see and edit in forms, among other things.
+
+There are two approaches to define entity types:
+- at runtime in the config file
+- at build time in the code
+
+Unless you have special requirements and write further code specifically for your entity type, 
+you usually should rely on the more flexible and easy approach defining them in the config. 
+
+
+## Defining a custom Entity type only through config
```

Comment:
I would rather not have this duplicate section here. Lets just have a link to the one in `configuration` and not having to maintain and update the info in two places.

Reply:
agreed. My tendency would be to use this "How To" guide to explain the details of entity config then and use the "Concepts" article about configuration only for the big picture approach of config + linking to this guide (+ linking to the interface .ts file specifying the available config props for entity configs - if we have that?).

What are your thoughts to disentangle this in the most readable way?

Reply:
As I understood it, this how to guide was about adding a new entity as actual code. Most of the content refers to code concepts. For people that only want to configure something I do not see this as a good starting point (they would have to read through a lot of stuff until we tell them that for doing it through the config they do not require any of that knowledge). 
So I would prefer having a very good documentation in the configuration section as this should hold all the required information that a administrator would need to adjust the config and keeping this guide solely for developers/contributors that want to extend the code. So I think a simple hint at the beginning that explain this (adding vs configuring entity) and refers to the config guide would make it easier for people to find the for them relevant place.

Reply:
We could also consider to move the part that explains the config file into a how to guide as it basically describes how to make adjustments to the app and split it from the implementation details that explain how the architecture behind loos like.

Reply:
I like that idea. Have added a stub for now and split the config part out of this guide :+1: